src/functionBased/components/InputTodo.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { FaPlusCircle } from 'react-icons/fa';
+import PropTypes from 'prop-types';
+
+const InputTodo = (props) => {
+  const [inputText, setInputText] = useState({
+    title: '',
+  });
+
+  const onChange = (e) => {
+    setInputText({
+      ...inputText,
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const { addTodoProps } = props;
+    if (inputText.title.trim()) {
+      addTodoProps(inputText.title);
+      setInputText({
+        title: '',
+      });
+    } else {
+      // alert('Please write item');
+    }
+  };
```

Comment:
Kindly make the button functional. It doesn't add the to-do item to the list when clicked on.

Reply:
Thanks for the review bro, I really appreciate it. 😊 
Cheers🥂